src/modules/visualization.js

```diff
@@ -92,22 +86,13 @@ export const transformVisualization = (visualization) => {
     }
 }
 
-const transformDimensions = (dimensions, { outputType, type }) =>
+const transformDimensions = (dimensions) =>
     dimensions.map((dimensionObj) => {
         if (dimensionObj.dimensionType === 'PROGRAM_DATA_ELEMENT') {
             return {
                 ...dimensionObj,
                 dimensionType: DIMENSION_TYPE_DATA_ELEMENT,
             }
-        } else if (
-            dimensionObj.dimension === DIMENSION_ID_PERIOD &&
-            type === VIS_TYPE_LINE_LIST
-        ) {
-            return {
-                ...dimensionObj,
-                dimension: outputTypeTimeDimensionMap[outputType],
-                dimensionType: DIMENSION_ID_PERIOD,
```

Comment:
But this line (109) should probably be removed, right?

Reply:
Apparently we do need `dimensionType: PERIOD` when saving, currently I get an error from the backend due to `pe` being passed as `dimensionType`.